package.json

```diff
@@ -8,12 +8,14 @@
     ]
   },
   "resolutions": {
-    "@polkadot/util-crypto": "7.9.2",
+    "@polkadot/util-crypto": "7.2.1",
```

Comment:
why are we downgrading?

out of curiosity why are all the polkadot packages on different versions? Don't they release them as a group?